cves/kernel/CVE-2020-25669.yml

```diff
@@ -55,7 +55,15 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: | 
+  Use-after-free pointer discovers in the Linux's keyboard drivers. 
```

Comment:
This first line is a little jarring. "Use-after-free pointer" makes sense, but I think "discovers" is supposed to be either "discovered" or "discovery made"

Reply:
Yes, It was supposed to be "discovered"; fixed.